bundles/org.openhab.binding.unifi/README.md

```diff
@@ -70,37 +94,123 @@ Additionally, you may use friendly site names as they appear in the controller U
 
 ##### `considerHome`
 
-The `considerHome` parameter allows you to control how quickly the binding marks a client as away. For example, using the default of `180` (seconds), the binding will report a client away as soon as `lastSeen` + `180` (seconds) < `now`
+The `considerHome` parameter allows you to control how quickly the binding marks a client as away.
+For example, using the default of `180` (seconds), the binding will report a client away as soon as `lastSeen` + `180` (seconds) < `now`.
+
+### `poePort`
+
+The following table describes the `poePort` configuration parameters:
+
+
+| Parameter   | Description                                                 | Config   |
+| ------------|-------------------------------------------------------------|----------|
+| portIdx     | The port number as reported by the switch                   | Required |
```

Comment:
Perhaps portNumber as name would be more straight-forward? Can it be mentioned in the description if they are numbered from 0 or 1?